.github/workflows/release-to-pypi.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4.3.0
         with:
-          python-version: 3.9
+          python-version: "3.10"
```

Comment:
Pyodide uses [Python 3.11](https://blog.pyodide.org/posts/0.23-release/) and the release candidate [Pyodide 0.26 are supporting Python 3.12](https://blog.pyodide.org/posts/0.23-release/) and [thread](https://github.com/pyodide/pyodide/issues/4420), so it might be worth building under those.

Reply:
First comment disappeared due to a Refresh hence the duplication, no idea why its come back from the dead.

Reply:
P.S. - Its only because `3.10` gets rounded to `3.1` if you'd jumped straight to `3.11` it would (I think) have worked ok. :smile:

Reply:
I think these versions are mainly just for building the wheel, so it just needed to be moved up to meet the minimum, but I can set it to 3.11 for now!

Separately, I can add 3.12 to the test matrix!

Reply:
Whoops! Yeah, very strange duplication of these threads? But yeah... I figured it was a rounding thing... I've kept it as a string so that nobody has to cry when version 3.20 rolls around :sweat_smile: